### DISCUSSION TITLE
Github runner: false 143 error with same code if __init__.py present in a package

### DISCUSSION THREAD

**Discussion by david-thrower** [community/community#54779] [Actions]
### Select Topic Area

Bug

### Body

I have a repo: `https://github.com/david-thrower/cerebros-core-algorithm-alpha`

Within it is a Python package /cerebros. It has several sub-folders within it, each having a `__init__.py` file. The root directory of the package `/cerebros` has no `__init__.py` file. It is should have one. I am trying to add it, but when I do, the tests I run fail **selectively when run in a github runner**. 

- If I run the tests in a GitHub runner without the `__init__.py` in the **root** directory of the package, the tests pass. 
- If I run them **with** the `__init__.py` file in the root directory of the package, on a bare metal Ubuntu 22.04 machine, it works. 
- If I run it  **with** the `__init__.py` file in the root directory of the package, in an equivalent Docker container, it works. 
- If I run it  **with** the `__init__.py` file in the root directory of the package, in an equivalent pod in Kubernetes, it works. 
- When it runs here by running these tests, it usually fails (143 or time out), but only if `__init__.py` is in the root directory of this package. (`https://github.com/david-thrower/cerebros-core-algorithm-alpha/blob/main/.github/workflows/automerge.yml`, more precisiely, this branch `https://github.com/david-thrower/cerebros-core-algorithm-alpha/blob/80-add-init-file-to-package-cerebros/.github/workflows/automerge.yml`), I get a 143 exit status ("Killed by an external signal") or a timeout. 

I notice in the logs an error that was caught, that the runner may be missing the fact that the error was caught:

2023-05-07T21:55:15.2508009Z 61/61 [==============================] - 153s 3s/step - loss: 0.0724 - top_1_categorical_accuracy: 0.9822 - val_loss: 1.6801 - val_top_1_categorical_accuracy: 0.7737

2023-05-07T21:55:23.2401801Z 2023-05-07 21:55:23.239661: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting **(this does not indicate an error and you can ignore this message)**: INVALID_ARGUMENT: You must feed a value for placeholder tensor 'inputs' with dtype float and shape [?,1,1,1280]

2023-05-07T21:56:27.9276506Z WARNING:absl:Function `_wrapped_model` contains input name(s)
2023-05-07T21:57:17.4714582Z 2023-05-07 21:57:17.467655: I tensorflow/core/common_runtime/executor.cc:1197] [/device:CPU:0] (DEBUG INFO) Executor start aborting **(this does not indicate an error and you can ignore this message)**: INVALID_ARGUMENT: You must feed a value for placeholder tensor 'inputs' with dtype float and shape [?,1,1,1280]

2023-05-07T21:57:17.9163424Z WARNING:absl:Found untraced functions such as _update_step_xla, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 65). These functions will not be directly callable after loading.
**2023-05-07T21:57:48.3258362Z ##[error]The runner has received a shutdown signal. This can happen when the runner service is stopped, or a manually started runner is canceled.
2023-05-07T21:57:48.3267419Z ##[error]Process completed with exit code 143.**
2023-05-07T21:57:48.4941201Z Cleaning up orphan processes

I'm completely puzzled with what is happening or why. Does anyone have any insight on this one?
